opentelemetry-exporter-gcp-trace/src/opentelemetry/exporter/cloud_trace/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.18.dev0"
+__version__ = "0.19.dev0"
```

Comment:
Missed this in a previous PR somehow

Reply:
Do we have tooling to automate editing this and/or enforce consistency across the different packages?